test/e2e/common.go

```diff
@@ -28,6 +28,13 @@ func Logf(format string, a ...interface{}) {
 	fmt.Fprintf(GinkgoWriter, "INFO: "+format+"\n", a...)
 }
 
+func checkError(err error) {
+	if err != nil {
+		fmt.Fprintf(GinkgoWriter, "ERROR: %v\n", err)
+		os.Exit(1)
```

Comment:
I'm not sure I would expect a function named `check...` to `os.Exit(1)`. I think this is more like `panic` than `check`. Does this exit the test process when the error is not nil?

I am not sure we need this function at all. Could we use `ginkgo.Fail` instead?

Reply:
Thanks @macaptain I have changed this function with `Expect(err).To(BeNil())`